Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+pipeline {
+    agent any
+
+    environment {
+
+	// These variables can be passed in a few different ways:
+	// 1. Set these here below.
+	// 2. Set them as "Parameterized" variables in Jenkins.
+	// 3. Set them as Environment variables from another process.
+	// Further reading on different ways to set environment variables https://jenkins.io/doc/pipeline/tour/environment/
+	APP_VERSION = "0.1.0"
+	REPLICATED_APP = ""
+
+	// If you use credentials in this file please encrypt them using this plugin https://wiki.jenkins.io/display/JENKINS/Credentials+Plugin
+	REPLICATED_API_TOKEN = ""
+    } 
+
+    stages {
+
+        stage('Fetch Replicated CLI') {
+            steps {
+                sh 'curl -fsSL https://github.com/replicatedhq/replicated/releases/download/v0.10.0/replicated_0.10.0_linux_amd64.tar.gz | tar xvz'
+            }
+        }
+
+        stage('Deploy App via CLI') {
```

Comment:
Does this actually deploy anything? What do you think about changing the title to something that describes that were creating / promoting a release?

Reply:
Yup it's just creating/promoting a release, will make the change.